config/initializers/paperclip.rb

```diff
@@ -0,0 +1 @@
+Paperclip::Attachment.default_options[:s3_host_name] = 's3-us-west-2.amazonaws.com'
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.